### DISCUSSION TITLE
Animation Transition to Frame 0 Starting Pose?

### DISCUSSION THREAD

**Discussion by najak3d** [stride3d/stride#1358] [Q&A] [answered]
I'm getting in some models with many animations.  And so to make these look better, without any modeling work, is there a way to tell Stride3D to "PrepareToStartAnimation" that automates a quick-timed transition where the model animates from current pose to the Animation's Frame 0 pose?   This prevents very obvious jitter that occurs otherwise.

Does Stride3D have any built-in methods to make this an easy thing to do?

**Comment by Eideren** [comment, marked as answer]
Follow this page of the manual to set the inital pose animation: https://doc.stride3d.net/4.0/en/Manual/animation/set-up-animations.html
Call `Crossfade` on that component with the animation you want to transition into.

**Reply by najak3d** [reply]
Thank you!   Stride3D wins again.  We're very much looking forward to converting our Urho3D.NET project over to Stride.
